README.md

```diff
@@ -29,13 +29,13 @@ If you're not sure if that's all installed or you don't know how then check out
 
 Using the standard go tools you can install Fyne's core library using:
 
-    $ go get fyne.io/fyne/v2
+    $ go install fyne.io/fyne/v2
```

Comment:
For the record, I think this an incorrect change. You still use `go get` to fetch and install dependencies for your modules.